### PR TITLE
feat(snomed): support expanding type() and destination() concepts on OWL Refset members

### PR DESCRIPTION
**Pull request by cmark**

### CODE REVIEW

**Review by nagyo — APPROVED** [review]
LGTM!

File: snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/index/entry/SnomedOWLRelationshipDocument.java [hunk 1]
```diff
@@ -68,6 +71,21 @@ public static SnomedOWLRelationshipDocument createFrom(final SnomedRelationship
...
+		} else {
+			return create(r.getTypeId(), r.getDestinationId(), r.getRelationshipGroup());
+		}
+	}
+	
+	public static List<SnomedOWLRelationshipDocument> createFrom(List<SnomedOWLRelationship> owLRelationships) {
+		if (owLRelationships == null) {
+			return null;
+		}
+		return owLRelationships.stream().map(SnomedOWLRelationshipDocument::createFrom).toList();
```
Comment (apeteri) [inline snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/index/entry/SnomedOWLRelationshipDocument.java:87]: `toList()` returns an unmodifiable list backed by an array. Is it acceptable for callers?

**Review by apeteri — APPROVED** [review]
🍃

File: snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/index/entry/SnomedOWLRelationshipDocument.java (hunk 1 above)
Comment (cmark) [reply snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/index/entry/SnomedOWLRelationshipDocument.java:87]: From the perspective of this method, yes. If a caller would like to modify this list it needs to wrap the contents in a mutable array and modify that.

**Pull request merged**